src/sst/core/configGraph.h

```diff
@@ -287,7 +287,10 @@ class ConfigComponent : public SST::Core::Serialization::serializable
     void setStatisticParameters(const std::string& statisticName, const Params& params, bool recursively = false);
     void setStatisticLoadLevel(uint8_t level, bool recursively = false);
 
-    void addGlobalParamSet(const std::string& set) { params.addGlobalParamSet(set); }
+    void                     addGlobalParamSet(const std::string& set) { params.addGlobalParamSet(set); }
+    std::vector<std::string> getParamsLocalKeys() const { return params.getLocalKeys(); }
+    std::vector<std::string> getSubscribedGlobalParamSets() const { return params.getSubscribedGlobalParamSets(); }
```

Comment:
Why are these copies instead of `const&`

Reply:
Because they are built only when the call is made.  They don't exist before or after so I don't have anything I can return a reference to.

Reply:
The underlying functions the functions call create the data structures on the fly, so there are no live variable to return a reference to.